db/migrate/20180724152308_create_reviews.rb

```diff
@@ -0,0 +1,18 @@
+class CreateReviews < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reviews do |t|
+      t.integer :overall_rating, null: false
+      t.text :body
+      t.integer :rides_rating
```

Comment:
if we validate for inclusion in the model, we should have these be `null: false` and / or set a reasonable default

Reply:
We want all the ratings except overall_rating to be optional. Should I use `numericality` instead? We do not want set `null: false` or a default constraint in the database-level.

Reply:
ok 👍